pkgs/tools/networking/curl/default.nix

```diff
@@ -5,8 +5,8 @@
 , zlibSupport ? true, zlib ? null
 , sslSupport ? zlibSupport, openssl ? null
 , gnutlsSupport ? false, gnutls ? null
-, scpSupport ? zlibSupport && !stdenv.isSunOS && !stdenv.isCygwin, libssh2 ? null
-, gssSupport ? !stdenv.hostPlatform.isWindows, libkrb5 ? null
+, scpSupport ? false, libssh2 ? null
+, gssSupport ? false, libkrb5 ? null
```

Comment:
I can change this to `!stdenv.isDarwin` if this is too controversial, but I don't think those should be enabled by default given that curlFull exists.